### PR TITLE
Made a route to get quizzes for a particular course and made somes fi…

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello-link : https://trello.com/c/uw7XZ3fG/87-course-detail-screen

### CODE REVIEW

File: server/routes/quiz.js [hunk 1]
```diff
@@ -13,7 +13,15 @@ router.post('/quiz/new', (req, res) => {
 });
 
 router.get('/quiz/quizById/:quizId', (req, res) => {
```
Comment (abdulhaqshah) [inline server/routes/quiz.js]: /quiz/:quizId

File: server/routes/quiz.js [hunk 2]
```diff
@@ -13,7 +13,15 @@ router.post('/quiz/new', (req, res) => {
...
-    quizLibs.getQuiz({quizId : req.params.quizId, active : true}).then((quiz) => {
+    quizLibs.getQuiz({quizId : req.params.quizId}).then((quiz) => {
+        res.status(quiz.status).send(quiz);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.get('/quiz/quizByCourse/:courseId', (req, res) => {
```
Comment (abdulhaqshah) [inline server/routes/quiz.js]: /quiz/course/:courseId

File: server/routes/tutorial.js [hunk 3]
```diff
@@ -13,15 +13,15 @@ router.post('/tutorial/new', (req, res) => {
 });
 
 router.get('/tutorial/tutorialById/:tutorialId', (req, res) => {
```
Comment (abdulhaqshah) [inline server/routes/tutorial.js]: /tutorial/:tutorialId

File: server/routes/tutorial.js [hunk 4]
```diff
@@ -13,15 +13,15 @@ router.post('/tutorial/new', (req, res) => {
...
-    tutorialLibs.getTutorial({tutorialId : req.params.tutorialId, active : true}).then((tutorial) => {
+    tutorialLibs.getTutorial({tutorialId : req.params.tutorialId}).then((tutorial) => {
         res.status(tutorial.status).send(tutorial);
     }).catch((error) => {
         res.status(error.status).send(error);
     });
 });
 
 router.get('/tutorial/tutorialByCourse/:courseId', (req, res) => {
```
Comment (abdulhaqshah) [inline server/routes/tutorial.js]: /tutorial/course/:courseId

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: server/routes/quiz.js (hunk 1 above)
Comment (HarisShahidHameed) [reply server/routes/quiz.js]: Done

File: server/routes/quiz.js (hunk 2 above)
Comment (HarisShahidHameed) [reply server/routes/quiz.js]: Done

File: server/routes/tutorial.js (hunk 3 above)
Comment (HarisShahidHameed) [reply server/routes/tutorial.js]: Done

File: server/routes/tutorial.js (hunk 4 above)
Comment (HarisShahidHameed) [reply server/routes/tutorial.js]: Done

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**